### ISSUE TITLE
domain whitelisting

### ISSUE THREAD

**Issue by RonnyDUC** [MetaMask/eth-phishing-detect#4244]
Our main website www.ducatus.com is flagged by Metamask because of its similar name to www.auctus.org. This is causing us a lot of unnecessary and unjustified problems and complaints.
My staff has been contacting you several times now to correct this issue but not even getting a response. 
I would like to ask you to look into this issue immediately please and resolve it by white labeling our website.

**Comment by muthunagarajan1** [comment]
Have raised another ticket at  https://metamask.zendesk.com/hc/en-us/requests/new with the following details.

The following are the links to issues & Pull requests raised to whitelist ducatus.com . These are open since 14th of August 2020.

https://github.com/MetaMask/eth-phishing-detect/pull/4276

https://github.com/MetaMask/eth-phishing-detect/issues/4105

https://github.com/MetaMask/eth-phishing-detect/issues/4105

You guys are actually harming genuine & legitimate businesses and sitting for months on raised concerns. Take corrective action.

**Comment by danfinlay** [comment]
Unblocked, users will stop seeing the warning as the cache expires. Sorry for falling behind on these, we are establishing processes for keeping up with the list in the future.

**Comment by muthunagarajan1** [comment]
Thank You!!
